.github/workflows/publishrpm.yml

```
@@ -0,0 +1,47 @@
+name: Publish rpm from master
+
+on:
+  schedule:
+    # every night at one
+    - cron: '0 1 * * *'
+  push:
+    tags:
+      - '*.*.*'
```

Maybe better this one?

```suggestion
      - '[0-9]+.[0-9]+.[0-9]+'
```

(Based on https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)